test/tests/test_harness/custom_eval

```diff
@@ -0,0 +1,7 @@
+[Tests]
```

Comment:
Please also test the cases where the user might supply `absent_out` or `expect_out` and an error is generated.